README.md

```diff
@@ -189,8 +189,9 @@ When a workflow (e.g. GitHub Action) or development event (e.g. pull request, co
 
 ## How the backfill works
 The app is designed to backfill historical data into Jira. Once you have installed and configured the app successfully,
-it will automatically trigger the backfilling process for the allowed repositories to update Jira with historical information such as pull requests, deployments, branches, builds, and commits.
+it will automatically trigger the backfilling process for 6 months for the allowed repositories to update Jira with historical information such as pull requests, deployments, branches, builds, and commits. Once the initial backfilling process is complete, you will be able to view the backfilled date and status on the user interface. All branches will be backfilled, regardless of their creation date. However, pull requests, deployments, builds, and commits will only be backfilled for the last six months. If you wish to pull more historical data in Jira, you may continue the backfill process for older dates by selecting 'Continue backfill' from the action menu. If the historical data is substantial, we recommend backfilling it in six-month slot and continuing the process until you've reached the desired backfilled date. 
```

Comment:
Edit: "...process for 6 months for the allowed..." to "...process, for 6 months, for the allowed..."